.dockerignore

```diff
@@ -0,0 +1,16 @@
+.circleci
+.github
+.gitignore
+.gitmodules
+.git
+.golangci.yml
+api
+app
+build/docker-images/
+dev
+docs
+examples
+go.mod
+go.sum
+Makefile*
+vendor
```

Comment:
That's really nice idea to add `.dockerignore`. I'd suggest also to add:
```
build/coverage/
pkg/**
!pkg/config/**
```

Reply:
Added, thanks for the suggestion!